apps/challenges/views.py

```diff
@@ -147,8 +147,8 @@ def add_participant_team_to_challenge(request, challenge_pk, participant_team_pk
 @api_view(['POST'])
 @permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
 @authentication_classes((ExpiringTokenAuthentication,))
+@permission_classes((IsChallengeCreator,))
 def disable_challenge(request, pk):
```

Comment:
Yup was working on that right now. 

Also, there are two `permission_classes` decorators. Do you want me to merge the two into one?